src/FSharpVSPowerTools.Logic/Navigation/GoToDefinitionFilter.fs

```diff
@@ -33,6 +28,12 @@ type internal UrlChangeEventArgs(url: string) =
     inherit EventArgs()
     member __.Url = url
 
+[<NoComparisonAttribute; NoEquality>]
+type GoToDefinitionResolution =
```

Comment:
Also `LoadDirective` clashes (potentially) with other places having this identifier (it is a DU in Untyped AST utils).